bin/poclcc.c

```diff
@@ -0,0 +1,188 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <CL/opencl.h>
+#include <assert.h>
+#include <string.h>
+
+char *kernelSource = NULL;
+char *outputFile = NULL;
+int openclDevice = CL_DEVICE_TYPE_DEFAULT;
+char *build_options = NULL;
+
+int print_help(){
+    printf("USAGE: poclcc [OPTION]... [FILE]\n");
+    printf("\n");
+    printf("OPTIONS:\n");
+    printf("\t--build_options=<options>\n");
+    printf("\t\tBuild the program with <options> options\n");
+    printf("\t-d <device_type>\n");
+    printf("\t\tSelect <device_type> as the device_type for clGetDeviceIDs." 
```

Comment:
It might be better to add selection to the pocl device id (starting from 0) and default to the first found device. In the future there might be multiple GPUs in the same platform etc.